flutter_screen_recording/lib/flutter_screen_recording.dart

```diff
@@ -1,37 +1,55 @@
+import 'dart:html' hide Platform;
+
 import 'package:flutter/foundation.dart';
 import 'package:flutter_screen_recording_platform_interface/flutter_screen_recording_platform_interface.dart';
 import 'dart:async';
 import 'dart:io';
 import 'package:flutter_foreground_plugin/flutter_foreground_plugin.dart';
 
 class FlutterScreenRecording {
-  static Future<bool> startRecordScreen(String name,
-      {String titleNotification, String messageNotification}) async {
+  static Future<bool> startRecordScreen(
+    String name, {
+    String titleNotification,
+    String messageNotification,
```

Comment:
If titleNotification and messageNotification are optional, you should set default value to it incase it's null

Reply:
Not show on web.

Reply:
Above code come from package author, We don't use that code so we do not need to maintain that now.